### ISSUE TITLE
Reactify not working in latest Chrome version

### ISSUE THREAD

**Issue by kentendo** [webcomponents/react-integration#58]
@treshugart 
Reactified component is rendering in Firefox, but not in latest Chrome (55.0.2883.95).
Component gets inserted inserted into DOM but is never upgraded and never renders itself.

See codepen below...
[http://codepen.io/kentendo/pen/BQMEbe?editors=1010](http://codepen.io/kentendo/pen/BQMEbe?editors=1010)

**Comment by kentendo** [comment]
I only see this happening in Chrome version 55.*
Works in 56.

**Comment by treshugart** [comment]
@kentendo do you have a reproduction of this? If you can write a failing test for this, that would be awesome :). Currently tests pass in 55.

**Comment by kentendo** [comment]
I started on a test, but looked at the other tests in here and noticed they are still using the v0 API and polyfill. The issue I'm seeing is more v1 specific, where the lifecycle hooks ( onConnectedCallback, etc )  are never getting called.  I'm afraid if I switch to a v1 polyfill, I'm going to have to rewrite all these old tests. Any ideas?

**Comment by treshugart** [comment]
You're right. We should update the tests, for sure. I raised #59 for this. I don't have much time to do this for the next week or so. Would you have time to submit a PR for it? I understand if not.

**Comment by kentendo** [comment]
I'll give it a shot...

**Comment by kentendo** [comment]
I've made some good progress on this, but I'm having issues with the event tests in chrome...
[https://github.com/kentendo/react-integration](https://github.com/kentendo/react-integration)
